nuxeo-jsf/nuxeo-web-resources-jsf/src/main/java/org/nuxeo/ecm/web/resources/jsf/AbstractResourceRenderer.java

```diff
@@ -79,6 +81,23 @@ protected String resolveUrl(FacesContext context, UIComponent component) throws
             String library = (String) attributes.get("library");
             url = resolveResourceUrl(context, component, library, name);
         }
+        return resolveUrlWithTimestamp(component, url);
+    }
+
+    protected String resolveUrlWithTimestamp(UIComponent component, String url) {
+        boolean doIncludeTimestamp = true;
+        Object includeTimestamp = component.getAttributes().get("includeTimestamp");
+        if (includeTimestamp instanceof String) {
+            if (!StringUtils.isBlank((String) includeTimestamp)) {
```

Comment:
I would like to keep the check to avoid resolving to false by default if the attribute is there but empty